src/time.rs

```diff
@@ -0,0 +1,8 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+
+pub fn create_timestamp() -> u64 {
+    let start = SystemTime::now();
+    let since_the_epoch = start.duration_since(UNIX_EPOCH)
+        .expect("Time went backwards");
+    return since_the_epoch.as_millis() as u64;
+}
```

Comment:
I'd  rather see this as a one-liner and return a Result: 

`pub fn create_timestamp() -> Result<u64> {
SystemTime::now().duration_since(UNIX_EPOCH)?.since_the_epoch.as_millis()
}
`

Reply:
This won't compile, but we can simplify the function

Reply:
just a nitpick, it's fine. I'm actually surprised a function like this does exist in std.